lms/models/application_settings.py

```diff
@@ -59,6 +62,44 @@ def set(self, group, key, value):
         """
         super().setdefault(group, {})[key] = value
 
+    def set_secret(self, aes_service, group, key, value: str):
```

Comment:
I reckon this belongs here, not in the admin pages around the code that deals with settings.